### PR TITLE
test: fix SimpleAggregate ranker

### PR DESCRIPTION
**Pull request by theUnkownName**

### CODE REVIEW

File: rankers/LevenshteinRanker/__init__.py [hunk 1]
```diff
@@ -17,13 +17,13 @@ class LevenshteinRanker(Match2DocRanker):
     required_keys = {"text"}
 
     def score(
-        self, query_meta: Dict, old_match_scores: Dict, match_meta: Dict
+        self, old_match_scores: Iterable[float], query_meta: Dict, match_meta: Iterable[Dict]
```
Comment (JoanFM) [inline rankers/LevenshteinRanker/__init__.py:20]: Levenshtein changes have already been taken care for

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: rankers/LevenshteinRanker/__init__.py (hunk 1 above)
Comment (theUnkownName) [reply rankers/LevenshteinRanker/__init__.py:20]: Yeah, I took your branch as it is so those changes were pushed too. I'm going to talk with Rutuja in half an hour on how to do the PRs for this

File: rankers/SimpleAggregateRanker/tests/test_aggregate_ranker.py [hunk 2]
```diff
@@ -143,12 +180,14 @@ def assert_document_order(doc_idx):
...
-        assert doc_idx == score
+        assert doc_idx[i][0] == str(doc_id)
+        assert doc_idx[i][1] == score
 
     ranker_deprecated = SimpleAggregateRanker(aggregate_function=aggregate_function, is_reversed_score=inverse_score)
-    doc_idx = ranker_deprecated.score(*chunk_scores)
+    doc_idx = fake_group_and_score(ranker_deprecated, *chunk_scores)
     assert_document_order(doc_idx)
     for i, (doc_id, score) in enumerate(zip(doc_ids, doc_scores)):
         assert int(doc_idx[i][0]) == doc_id
```
Comment (JoanFM) [inline rankers/SimpleAggregateRanker/tests/test_aggregate_ranker.py:193]: the way of iteration here is weird

Comment (JoanFM) [reply rankers/SimpleAggregateRanker/tests/test_aggregate_ranker.py:193]: would it make more sense:

```
for doc_id, score in zip(doc_ids, doc_scores))
     assert score == doc_id
```
?

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: rankers/SimpleAggregateRanker/tests/test_aggregate_ranker.py (hunk 2 above)
Comment (Yongxuanzhang) [reply rankers/SimpleAggregateRanker/tests/test_aggregate_ranker.py:191]: Why assert score == doc_id?

**Review by JoanFM — APPROVED** [review]

File: rankers/SimpleAggregateRanker/tests/test_aggregate_ranker.py (hunk 2 above)
Comment (JoanFM) [reply rankers/SimpleAggregateRanker/tests/test_aggregate_ranker.py:192]: forget it

**Comment by Yongxuanzhang** [comment]
So we just fake the group and sort function. Then all tests will remain the same as before. Make sure that we can still maintain the behaviour.

**Review by JoanFM — APPROVED** [review]

**Pull request merged**